manifests/p/py2exe/py2exe/0.6.9/py2exe.py2exe.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+PackageIdentifier: py2exe.py2exe
+PackageVersion: 0.6.9
+PackageLocale: en-US
+Publisher: py2exe
+Author: Thomas Heller, Jimmy Retzlaff, Mark Hammond, Alberto Sottile
+PackageName: py2exe

```

```suggestion
PackageName: py2exe
PackageUrl: https://www.py2exe.org/
```

Adding the package url helps contributors to better handle this package :)